### PR TITLE
feat: Re-invite participants when a bridge goes to SHUTTING_DOWN.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/main/java/org/jitsi/jicofo/bridge/JvbDoctor.java [hunk 1]
```diff
@@ -129,6 +130,11 @@ public void bridgeAdded(Bridge bridge)
         logger.info("Scheduled health-check task for: " + bridge);
     }
 
+    @Override
+    public void bridgeIsShuttingDown(@NotNull Bridge bridge)
```
Comment (JonathanLennox) [inline src/main/java/org/jitsi/jicofo/bridge/JvbDoctor.java:134]: Why is this needed? Does the default Kotlin implementation not apply to Java implementations of interfaces?

Comment (bgrozev) [reply src/main/java/org/jitsi/jicofo/bridge/JvbDoctor.java:134]: The kotlin function needs a `@JvmDefault` annotation, but that requires to run with `-Xjvm-default` and I didn't want to do that.

**Review by JonathanLennox — APPROVED** [review]
